### PR TITLE
Add no Cancel button option

### PR DESCRIPTION
**Pull request by Deuteu**
Issue #33

### CODE REVIEW

File: jquery.confirm.js [hunk 1]
```diff
@@ -101,6 +101,13 @@
                     '<h4 class="modal-title">' + settings.title+'</h4>' +
                 '</div>';
         }
+        var cancelButtonHtml = '';
+        if (settings.cancelButton) {
```
Comment (mnapoli) [inline jquery.confirm.js]: By default the property will not exist right? So the cancel button will disappear when people upgrade

Comment (mnapoli) [inline jquery.confirm.js]: Oh wait you are re-using the "cancel button" label option :) How about updating the documentation of that option in the README: https://github.com/myclabs/jquery.confirm#options ?

Comment (Deuteu) [inline jquery.confirm.js]: Oh yeah indeed, forgot about that.
Will do it asap.

**Comment by Deuteu** [comment]
@mnapoli Should be good now.

**Comment by mnapoli** [comment]
Nice thanks!

**Pull request merged**